### PR TITLE
Add synternet

### PR DESCRIPTION
**Pull request by auridasf**

### CODE REVIEW

File: synternet/chain.json [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      },
+      {
+        "id": "28d36c3d45f0208528de3c38f2934ae241bd23e7",
+        "address": "peer-cosmoshub.mms.team:26656",
+        "provider": "MMS"
+      },
+      {
+        "id": "87ccc1dcc0b846fc1623ab9a5ab55682e8e2ad2e",
+        "address": "seed-cosmoshub.freshstaking.com:26656",
+        "provider": "FreshSTAKING"
```
Comment (auridasf) [inline synternet/chain.json]: @VJurenas who communicates with our validators? We need to get this information from them

Comment (VJurenas) [reply synternet/chain.json]: 2eed7e175d204680af243e008e21950f81b8d455 @ 34.89.206.173:26656 (Synternet)
cab3e0b0538e900d57a83ad590a3d228004c63a4 @ 35.225.165.111:26656 (Stakin)
6f76ec0b365da744e893b470eac72a87482359d5 @ 176.9.125.13:2220 (Imperator)
ac6ec924a0501b55db82c6a76aab893aed9a8630 @ 65.109.122.249:26656 (Polkachu)
95b14ec701608044c261ebd15d0b3bd84e295acb @ 72.46.84.135:26656 (Restake)

Comment (auridasf) [reply synternet/chain.json]: Thanks, updated

**Review by foxis — APPROVED** [review]
LGTM

**Pull request merged**